src/frontend/screens/ObservationFields/Number.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {StyleSheet, TextInput} from 'react-native';
+import {QuestionLabel} from './QuestionLabel';
+import {Field} from '@comapeo/schema';
+import {usePersistedDraftObservation} from '../../hooks/persistedState/usePersistedDraftObservation';
+import {useDraftObservation} from '../../hooks/useDraftObservation';
+
+export const Number = React.memo<{field: Field}>(({field}) => {
+  const tags = usePersistedDraftObservation(store => store.value?.tags);
+  const {updateTags} = useDraftObservation();
+  const value = tags ? tags[field.tagKey] : '';
+  return (
+    <React.Fragment>
```

Comment:
should there be/ can there be a minimum and/ or maximum? And is it intended that negative numbers and decimals aren't allowed?